apps/web/middleware.ts

```diff
@@ -183,13 +193,11 @@ export const config = {
     "/routing-forms/:path*",
     "/team/:path*",
     "/org/:path*",
-    "/:user/:type/",
-    "/:user/",
   ],
 };
 
 export default collectEvents({
-  middleware: abTestMiddlewareFactory(middleware),
```

Comment:
abTestMiddlewareFactory isn't used and needed (it was testing infra for app dir when we supported /future dir)